tests/signed-off-by-test.sh

```diff
@@ -4,10 +4,9 @@ success="true"
 
 for commit in $(git cherry master | cut -d " " -f 2)
 do
-    git show -s $commit | grep -q 'Signed-off-by:'
-    if [ $? -ne 0 ]; then
+    if ! git show -s "$commit" | grep -q 'Signed-off-by:'; then
```

Comment:
I meant separate pull request rather than commit (my bad). Once that's opened we can merge that first.